<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMainForm</class>
 <widget class="QWidget" name="QMainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Block Blast</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QFrame" name="frameFigure1">
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>155</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="figureArea1">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>120</width>
          <height>120</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonPrev">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;-</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonSig">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>70</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>-&gt;</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frameFigure2">
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>155</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="figureArea2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>120</width>
          <height>120</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frameFigure3">
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>155</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="figureArea3">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>120</width>
          <height>120</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
      <widget class="QFrame" name="containerArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>480</width>
         <height>480</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>420</width>
         <height>595</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Puntuación</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonComenzar">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>90</y>
          <width>181</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Parar</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBoxPruebas">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Modo de pruebas</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <widget class="QPushButton" name="buttonVaciar">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Vaciar contenedor</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonLineasCompletas">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>80</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Lineas completas</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonBorrarCeldas">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>80</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Borrar celdas´</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonInsertarF">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>40</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Insertar figura</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonBorrarF">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>40</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Borrar figura</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonHuecoLibre">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Hueco libre</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonPosiblJugada">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>141</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Posible jugada</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxColorFijo">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>120</y>
          <width>92</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Color fijo</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxColores">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>120</y>
          <width>131</width>
          <height>25</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Azul</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Verde</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rojo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Amarillo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rosa</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Naranja</string>
         </property>
        </item>
       </widget>
       <widget class="QFrame" name="framePosibleJ">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>120</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>groupBoxPruebas</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonPrev</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBoxPruebas</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonSig</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>634</y>
    </hint>
    <hint type="destinationlabel">
     <x>738</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
